app/controllers/items_controller.rb

```diff
@@ -4,9 +4,13 @@ def new
     type = params[:type]
     druid = params[:druid]
     targets = params[:targets]
+    raise 'missing required params' unless type && druid && targets
```

Comment:
Does this work when you aren't using the params in a model object though?  I thought it had do with mass assignment and not when you are just using the values to pass to a method.